src/config/config.js

```diff
@@ -190,5 +190,24 @@ export default {
    * @memberof module:config
    * @property {Array} known_notations
    */
-  known_notations: ["preferred-email-encoding@pgp.com", "pka-address@gnupg.org"]
+  known_notations: ["preferred-email-encoding@pgp.com", "pka-address@gnupg.org"],
+  /**
+   * use external indutny/elliptic library
+   */
+  external_indutny_elliptic: false,
+  /*
+   * use indutny/elliptic library
+   */
+  use_indutny_elliptic: true,
+  /**
+   * Path to load elliptic from external file
+   */
+  external_indutny_elliptic_path: 'elliptic.min.js',
+
+  /*
+   * indutny elliptic fetch options
+   */
+  indutny_elliptic_fetch_options: {
+    credentials: 'include'
+  }
```

Comment:
This default `credentials: 'include'` is not necessary, it can be set by the application if needed. You can just default to `{}` for the options, I think.

Reply:
fixed